prisondersdilemma.py

```diff
@@ -4,199 +4,438 @@
+						   wrapWidth=400)
+	#instruction text
+	inststr = f'''
+	Imagine you are one of two people being questioned about the same crime. They are each talking 
+	to the interrogator separately. The interrogator gives each person the same deal: 
+	they can choose to vouch for the other person’s innocence (COOPERATING) or rat them out (DEFECTING). 
+	And of course, there’s a twist. 
+	If both people cooperate with each other, they’ll each get 3 months off their sentence, 
+	but if the first person vouches for the second person, and the second person rats them out, 
+	the first person will get no time off their sentence and the second person will get 5 months off their time. 
+	Lastly, if they both rat each other out, they each get 1 month off their time.
+	
+	Press 'space' to start the game. Press 'Q' to exit early.
+	'''
+	
+	insttxt = TextStim(win, text=inststr, font='Arial', height=20, color='black', 
+						   wrapWidth=400)
+	
+	#objects for representing participants, images
+	#telling people to choose avatar
+	avatstr = f'''
+	Please choose one of these avatars to represent you in the game.
+	Just press the key that responds to letter next to the image.
+	'''
+	avattxt = TextStim(win, text=avatstr, font='Arial', height=20, color='black', 
+						   wrapWidth=400, pos=(000, -300))
+	
+	#avatars: created with http://www.hexatar.com
+	#opponent image stimuli, there are on the right of the screen
+	opponent = ImageStim(win, image='opponent2.png', pos=(300, 000)) #placeholder
+	participant = ImageStim(win, image='partA', pos=(-300, 000)) #placeholder
+	oppcauc = ImageStim(win, image='opponent2.png', pos=(300, 000)) #opponent is caucasian ethnicity 
+	oppnoncauc = ImageStim(win, image='opponent1.png', pos=(300, 000)) #opponent who's not caucasian
+	oppunknown = ImageStim(win, image='anonymous.jpg', pos=(300, 000))
+	
+	#participant representation, they are on the left side of the screen
+	partA = ImageStim(win, image='partA.png', pos=(-300, 000)) #participant female caucasian
+	partB = ImageStim(win, image='partB.png', pos=(-300, 000)) #participant female non caucasian
+	partC = ImageStim(win, image='partC.png', pos=(-300, 000)) #participant male caucasian
+	partD = ImageStim(win, image='partD.png', pos=(-300, 000)) #participant male non caucasian
+	partoverview = ImageStim(win, image='partchoice.png') #start screen from which participant chooses an avatar to represent them in the game
+	
+	#objects for during the game, in the loop
+	sentcountp = 35 #sentence count for the participant
+	sentcounto = 35 #sentence count for the opponent
+	
+	#info about controls/gameplay, should be displayed on the bottom left during the game
+	controlinfo = TextStim(win, text='''Press the left arrow to defect 
+						   and the right arrow to cooperate''',
+						   font='Arial', height=20, color='black',
+						    pos=(-300, -300))
+	
+	# txt within game
+	connectstr = 'Your prison sentence is' + str(sentcountp) + 'months.\n' +
```

Comment:
If you put it between brackets, maybe it works :) 
connectstr = ('Your prison sentence is' + str(sentcountp) + 'months.\n' +
	'Your partner`s sentence is' + str(sentcounto) + 'months.\n' +
	'The authorities are still not sure what to do with the two of you. Therefore, you and your partner\n'
	'will be interrogated again by a different police officer. Again you have the choice to cooperate or defect.')